DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -176,6 +185,14 @@ bool rememberMe
                 IssuedUtc = DateTime.UtcNow
             };
             HttpContext.SignInAsync("Identity.Application", new ClaimsPrincipal(claimsIdentity), authProperties);
+            if (!string.IsNullOrEmpty(returnUrl))
+            {
+                if (Url.IsLocalUrl(returnUrl))
+                {
+                    return Redirect(returnUrl);
+                }
+                logger.LogWarning($"Attempted login redirect to non-local returnUrl {returnUrl}");
```

Comment:
Does this work ok with the old site (they're on the same baseUrl so should be ok)?

Reply:
I think so but we should test it on UAT as there may be some trouble depending on how the absolute URLs are constructed.